src/config/PluginConfigManager.ts

```diff
@@ -110,7 +110,7 @@ export class PluginConfigManager<TPluginType extends BasePluginType> {
     const level = matchParams?.level ?? (member && this.getMemberLevel(member)) ?? null;
 
     // Passed roles -> passed member's roles
-    const memberRoles = matchParams.memberRoles || (member && member.roles.cache.keyArray());
+    const memberRoles = matchParams.memberRoles ?? member?.roles?.cache?.keyArray() ?? [];
```

Comment:
Unnecessary optional chaining
```suggestion
    const memberRoles = matchParams.memberRoles ?? member?.roles.cache.keyArray() ?? [];
```

Reply:
It isn't, similarly to [this comment](https://github.com/Dragory/Knub/pull/10#discussion_r663560518) something causes this to be called once with an object that exists but doesnt have roles, but then it also crashed since cache didnt exist. I have no clue what is causing this, this is a bandaid fix that needs to be investigated deeper by @Dragory